app/components/notifications-dropdown/index.scss

```diff
@@ -1,10 +1,11 @@
 .notification-dropdown-container {
-  position: relative;
+  position: absolute;
   width: 35em;
   background: white;
   border: 1px #f1f1f1 solid;
   box-shadow: 0 5px 20px 0px rgba(0, 0, 0, 0.1);
-  margin-right: 2em;
+  transform: translateX(-66%);
+  top: 10px;
```

Comment:
this is not required please refer `ak-menu` implementation to fix alignment issue